src/Microsoft.AspNetCore.Mvc.Core/Internal/MvcCoreLoggerExtensions.cs

```diff
@@ -143,10 +145,10 @@ internal static class MvcCoreLoggerExtensions
 
         static MvcCoreLoggerExtensions()
         {
-            _actionExecuting = LoggerMessage.Define<string>(
-                LogLevel.Debug,
+            _actionExecuting = LoggerMessage.Define<StringBuilder, string>(
+                LogLevel.Information,
                 1,
-                "Executing action {ActionName}");
+                "Route {Route} matched. Executing action {ActionName}");
```

Comment:
Suggestion, change this to `"Route matched with {RouteData}. Executing action {ActionName}"`